coops-ui/package.json

```diff
@@ -13,9 +13,10 @@
   },
   "dependencies": {
     "@babel/polyfill": "^7.0.0-rc.1",
+    "@types/vuelidate": "^0.7.5",
```

Comment:
Should Vuelidate version be updated also?
I now the ^ means minor versions are updated, but visually matching the versions would look consistent.

Reply:
Oh, good catch! I'll drop the types/vuelidate to 0.7.4, since that's what version of vuelidate I'm actually running (there is no vuelidate 0.7.5).